packages/components/alert/src/alert.vue

```diff
@@ -91,6 +92,11 @@ const { onOpen, onClose } = useDelayedToggle({
 })
 
 if (isClient) {
-  onOpen()
+  const rawProps = getCurrentInstance()?.vnode?.props ?? {}
+  const immediate = ['showAfter', 'show-after', 'show-After'].every(
+    (key) => !hasOwn(rawProps, key)
+  )
+
+  onOpen(undefined, immediate)
```

Comment:
In order to not surcharge the delayed toggle hook maybe you can consider this change:

```suggestion
  if(immediate) {
    open()
  }
  onOpen()
```

We should not involve adhoc solution in external/hook code if possible.

Reply:
In that case, it seems the original problem has resurfaced.😄

Reply:
Nice, well done :+1: